bitnami/nginx/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx
-version: 6.0.2
+version: 6.0.3
```

Comment:
Please bump minor version instead

Reply:
Thanks for the review 
I have bumped the version to 6.1.0 as a minor version